qtconsole/mainwindow.py

```diff
@@ -730,6 +739,20 @@ def toggleFullScreen(self):
     def set_paging_active_frontend(self, paging):
         self.active_frontend._set_paging(paging)
 
+    def get_available_syntax_styles(self):
```

Comment:
We can use a pygments function to find the available styles:
http://pygments.org/docs/styles/#getting-a-list-of-available-styles